netlify.toml

```diff
@@ -1,3 +1,7 @@
 [[redirects]]
   from = "/introduction/contributing/"
   to = "/contribute/"
+
+[[redirects]]
```

Comment:
Nice, I would have forgotten about that...

Reply:
I probably too If I hadn't managed many website releases the last couple of weeks were redirects and Google Search Console were my primary tools :)